src/router.ts

```diff
@@ -61,7 +63,7 @@ export function createRouter (opts: CreateRouterOptions = {}): Router {
     // Match route
     const matched = _router.lookup(path);
     if (!matched || !matched.handlers) {
-      if (opts.preemtive) {
+      if (opts.preemptive || opts.preemtive) {
```

Comment:
Do not taint the logic code with hacks like this

A better solution would be preprocessing the options
`opts.preemptive ||= opts.premtive`
right when the function starts

Reply:
This is good. We also avoid using recent syntaxes (logical or supported for Node 15+).